servers/rendering/renderer_rd/renderer_canvas_render_rd.cpp

```diff
@@ -676,16 +688,21 @@ void RendererCanvasRenderRD::_render_item(RD::DrawListID p_draw_list, RID p_rend
 
 				_bind_canvas_texture(p_draw_list, polygon->texture, current_filter, current_repeat, last_texture, push_constant, texpixel_size);
 
-				push_constant.modulation[0] = base_color.r;
-				push_constant.modulation[1] = base_color.g;
-				push_constant.modulation[2] = base_color.b;
-				push_constant.modulation[3] = base_color.a;
+				Color color = base_color;
+				if (use_linear_colors) {
+					color = color.srgb_to_linear();
+				}
+
+				push_constant.modulation[0] = color.r;
+				push_constant.modulation[1] = color.g;
+				push_constant.modulation[2] = color.b;
+				push_constant.modulation[3] = color.a;
 
 				for (int j = 0; j < 4; j++) {
 					push_constant.src_rect[j] = 0;
 					push_constant.dst_rect[j] = 0;
 					push_constant.ninepatch_margins[j] = 0;
-				}
+				} //CLAY
```

Comment:
![image](https://github.com/godotengine/godot/assets/4701338/2bce55c5-9414-4534-9781-e42a081655b2)

Reply:
Whoops! I use this instead of "TODO" so I can easily find my place when switching locations within a file.